restrict_registration/language/en-GB/en-GB.plg_user_restrictregistration.ini

```diff
@@ -0,0 +1,4 @@
+PLG_USER_RESTRICT_REGISTRATION="Restrict user registration"
+PLG_USER_RESTRICT_REGISTRATION_USER_COUNT_LABLE="User Restrict Count"
+PLG_USER_RESTRICT_REGISTRATION_USER_COUNT_LABLE_DESC="Enter user restrict count"
```

Comment:
PLG_USER_RESTRICT_REGISTRATION_USER_COUNT_LABLE="Max Allowed Registrations Count"
PLG_USER_RESTRICT_REGISTRATION_USER_COUNT_LABLE_DESC="Set count of max no. of users that can be registered on website"
PLG_USER_RESTRICT_REGISTRATION_MSG="Cannot add more users. Max no. of allowed registrations count is reached."

Reply:
updated